core/modules/modulesconf.py

```diff
@@ -0,0 +1,46 @@
+from viur.core.bones import StringBone
+from viur.core.tasks import StartupTask
+from viur.core import conf, db
+from viur.core.i18n import translate
+from viur.core.skeleton import Skeleton, SkeletonInstance
+from viur.core.prototypes import List, BasicApplication
+
+
+class ModulesConfSkel(Skeleton):
+    kindName = "viur-modules-conf"
```

Comment:
I would like to recommend a singluar naming? As this is the kind defining conf for a module each entry.
```suggestion
    kindName = "viur-module-conf"
```